Baremetal-Blinky/mbed-os.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-os-ti-port/#1304ebdb5e8d1e6d13aabe1360813b75f58c0828
```

Comment:
Woah, interesitng.

Will be nice to see this eventually updated to ARMmbed/mbed-os one day 😄